ui/app/AppLayouts/Wallet/controls/StatusNetworkSelector.qml

```diff
@@ -0,0 +1,224 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14 as QC
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1
+
+import utils 1.0
+
+/*!
+   \qmltype StatusNetworkSelector
+   \inherits Rectangle
+   \inqmlmodule StatusQ.Components
+   \since StatusQ.Components 0.1
+   \brief It allows to add items and display them as a tag item with an image and text. It also allows to store and display logical `and` / `or` operators into the list. Inherits \l{https://doc.qt.io/qt-6/qml-qtquick-rectangle.html}{Item}.
+
+   The \c StatusNetworkSelector is populated with a data model. The data model is commonly a JavaScript array or a ListModel object with specific expected roles.
+
+   Example of how the component looks like:
+   \image status_item_selector.png
+
+   Example of how to use it:
+   \qml
+        StatusNetworkSelector {
+            id: networkSelector
+
+            title: "Network preference"
+            enabled: addressInput.valid
+            defaultItemText: "Add networks"
+            defaultItemImageSource: "add"
+
+            itemsModel: ListModel {}
+
+            addButton.onClicked: {
+            }
+
+            onItemClicked: {
+            }
+
+            onItemRightButtonClicked: {
+            }
+        }
+   \endqml
+   For a list of components available see StatusQ.
+*/
+Rectangle {
+    id: root
```

Comment:
Isn't this a good candidate to be part of `StatusQ`?

Reply:
@IvanBelyakoff as usual there is always a solution ;) if we decide to add it to `StatusQ` means that`StatusNetworkSelector` needs to know how to display supported networks, that further means icons for supported networks should be there as well. So you can add all necessary icons to statusq.

Another way is to provide resolved icon path via `iconUrl` model's role.